Data/Aeson/Parser/Internal.hs

```diff
@@ -2,12 +2,9 @@
 {-# LANGUAGE CPP #-}
 {-# LANGUAGE NoImplicitPrelude #-}
 {-# LANGUAGE OverloadedStrings #-}
-#if MIN_VERSION_ghc_prim(0,3,1)
-{-# LANGUAGE MagicHash #-}
-#endif
 #if __GLASGOW_HASKELL__ <= 710 && __GLASGOW_HASKELL__ >= 706
 -- Work around a compiler bug
-{-# OPTIONS_GHC -fsimpl-tick-factor=200 #-}
+{-# OPTIONS_GHC -fsimpl-tick-factor=300 #-}
```

Comment:
I had to increase the limit because CI fails on ghc-7.8.4 with `Simplifier ticks exhausted`: https://travis-ci.org/bos/aeson/jobs/653523217?utm_medium=notification&utm_source=github_status
Note that this affects only older ghc versions.